repobee_sanitizer/sanitizer.py

```diff
@@ -82,10 +85,15 @@ def command(self) -> Optional[plug.Result]:
         else:
             LOGGER.info(f"Sanitizing repo and updating {self.target_branch}")
             try:
+                if self.create_pr_branch:
+                    pr_branch_name = "sanitizer-pull-request"
```

Comment:
I rewrote the code to use a variable with this in mind so that we can just add a timestamp or similar to this once we've changed some other, more urgent, things first (such as much of the feedback you've given so far)

Reply:
@slarse Sanitizer just force adds everything to the target branch, which would be the PR branch, i can't see any issues with using the same branch name, it's what we've been doing all along when sanitizing to master at least functionality-wise.

Unless you mean that the problem is inside the fact we want everything before we open the PR to be the commit history from target_branch and not the branch we sanitize from.

Reply:
Either way it doesnt really matter, i agree that a new branch everytime is better, just not sure if necessary, so we can just implement this anyway

Reply:
> @slarse Sanitizer just force adds everything to the target branch, which would be the PR branch, i can't see any issues with using the same branch name, it's what we've been doing all along when sanitizing to master at least functionality-wise.

This will cause merge conflicts if you squash or rebase into master. Try doing it manually to see what happens.

1. Create a repo with an arbitrary commit on the master branch.
2. Check out to a "PR" branch.
3. Make any amount of commits on the "PR" branch
4. Squash merge "PR" into master
5. Make another commit on "PR"
6. Squash merge "PR" into master ---> conflict!

There are many reasons why branch reuse isn't a good idea, this is one of them.